src/libs/actions/IOU.ts

```diff
@@ -2248,6 +2248,35 @@ function trackExpense(
     Report.notifyNewAction(activeReportID, payeeAccountID);
 }
 
+function getOrCreateOptimisticSplitChatReport(existingSplitChatReportID: string, participants: Participant[], participantAccountIDs: number[], currentUserAccountID: number) {
```

Comment:
@marcaaron thank you for your prompt PR 🙇 I have one question 😄 

Did you create the new function, getOrCreateOptimisticSplitChatReport, solely for code refactoring? Or was this change necessary to resolve the original issue?

Reply:
I didn't see https://github.com/Expensify/App/issues/39247 and https://github.com/Expensify/App/issues/39254. I'm sorry about that 🙇